assets/messages.yml

```diff
@@ -1295,22 +1168,23 @@ messages:
         * The feature behaves differently in one edition than in the other
         * The parity issue was introduced in Buzzy Bees (Bedrock Edition 1.14 / Java Edition 1.15) or later and was not present before
 
-        Any parity issue that does not meet these criteria will not be tracked on the bug tracker and should instead be reported on the [Minecraft Feedback Discord server|https://aka.ms/MinecraftFeedbackDiscord/].
+        Any parity issue that does not meet these criteria will not be tracked on the bug tracker and should instead be reported on the [Minecraft Feedback Discord server|https://aka.ms/MinecraftFeedbackDiscord].
 
         %quick_links%
       fillname: []
   pirated-minecraft:
     - project:
         - mc
         - mcl
+        - mcpe
```

Comment:
Is it worth adding WEB here? Sometimes people report pirated things there. It's rare, but can happen

Reply:
Wouldn't hurt for sure. I'm not sure myself what warrants a message not being enabled for a certain project if it is at least somewhat applicable to it.

Reply:
I'm happy to add it even if it's minimal usage. Doesn't do any harm and WEB doesn't have so many messages that it'd cause clutter either